ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/numberinput.html

```diff
@@ -29,7 +29,7 @@
      data-cmp-enabled="${numberinput.enabled ? 'true' : 'false'}"
      id="${numberinput.id}"
      data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
-    <div data-sly-call="${label.label @componentId=numberinput.id, labelValue=numberinput.label.value, labelVisible=numberinput.label.visible, bemBlock='cmp-adaptiveform-numberinput'}"></div>
+    <div data-sly-call="${label.label @componentId=numberinput.id, labelValue=numberinput.label.value, labelVisible=numberinput.label.visible, bemBlock='cmp-adaptiveform-numberinput'}" data-sly-unwrap></div>
```

Comment:
but we are not unwrapping label in any other component..

Reply:
@sakshi-arora1 we are doing.
https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html#L32

Reply:
So we need to fix in all, datepicker doesn't. Lets scan all components and add it where ever missing.

Reply:
I have checked all the components and fixed them.

Reply:
https://github.com/adobe/aem-core-forms-components/blob/dev/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.html#L31 
unwrap is not present here

Reply:
I added it in the last commit of this PR

Reply:
ok